.github/workflows/release.yml

```diff
@@ -129,7 +132,36 @@ jobs:
         with:
           name: azureauth-${{ github.event.inputs.version }}-${{ matrix.runtime }}
           path: azureauth-${{ github.event.inputs.version }}-${{ matrix.runtime }}
-
+  
+  # Build and sign Linux binaries on Azure DevOps and publish them to GitHub and packages.microsoft.com.
+  linux_build_and_sign:
+    runs-on: ubuntu-latest
+    needs: [validate]
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Setup Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.10'
+    - name: Build, Sign and Download Linux Binaries
+      run: |
+        pip install -r bin/requirements.txt
+        python ./bin/trigger_azure_pipelines.py
+      env:
+        AZURE_DEVOPS_BUILD_PAT: ${{ secrets.AZURE_DEVOPS_BUILD_PAT }}
+        ADO_ORGANIZATION: ${{ secrets.ADO_ORGANIZATION }}
+        ADO_PROJECT: ${{ secrets.ADO_PROJECT}}
+        ADO_AZUREAUTH_LINUX_PIPELINE_ID: ${{ secrets.ADO_AZUREAUTH_LINUX_PIPELINE_ID }}
+        VERSION: ${{ github.event.inputs.version }}
+        ADO_LINUX_ARTIFACT_NAME: ${{ secrets.ADO_LINUX_ARTIFACT_NAME }}
+        ADO_LINUX_ARTIFACT_DOWNLOAD_PATH: dist/linux-x64

```

Comment:
Given this value (`ADO_LINUX_ARTIFACT_DOWNLOAD_PATH`) is hardcoded here, to change it would require a PR updating the pipeline. So, effectively it's not much different if the value were hardcoded in the script directly. It's no more difficult to change it - however, having the value in the script makes the script easier to understand beacuse you can see what that path is going to be.

Reply:
Though I can also see you may have opted for putting it here, since it is used in the step below, and this let's you only define it in these 2 pipeline steps...

Pipeline are notoriosly easy to break, so to avoid that I would avoid hardcoding the value more than once, and declare this env var at the `Job` level and reference the env var still from the script, but also reference the env var in the upload step below.

Reply:
Understood. I will use this as an env var at the job level.